workitem/link/link_repository.go

```diff
@@ -40,6 +40,7 @@ type WorkItemLinkRepository interface {
 	Save(ctx context.Context, linkCat WorkItemLink, modifierID uuid.UUID) (*WorkItemLink, error)
 	ListWorkItemChildren(ctx context.Context, parentID uuid.UUID, start *int, limit *int) ([]workitem.WorkItem, uint64, error)
 	WorkItemHasChildren(ctx context.Context, parentID uuid.UUID) (bool, error)
+	GetParentID(ctx context.Context, ID uuid.UUID) (*uuid.UUID, error)
```

Comment:
Please, move the comment from the function to this interface here.

Reply:
linter advise to keep the comment for exported functions hence I am keeping it at both locations